src/Components/ArtworkGrid/LazyArtworkGrid.tsx

```diff
@@ -0,0 +1,18 @@
+import StaticContainer from "found/StaticContainer"
+import { useRouter } from "found"
+
+/**
+ * When loading artwork grids / filter lazily we need to ensure that rerenders
+ * don't occur when the route changes, which then triggers the skeleton loading
+ * state. This component will prevent rerenders when the route changes, signalled
+ * by `match.elements` being undefined per the router.
+ */
+export const LazyArtworkGrid: React.FC = ({ children }) => {
```

Comment:
I don't _love_ the 'LazyArtworkGrid' naming as there's nothing 'lazy' about _this_ component at all, it's more about wrapping an artwork grid that is behind a query renderer / lazy query. Maybe 'StaticArtworkGrid' or something is better?

Although, that winds up maybe being not super intuitive, vs 'oh this artwork grid is lazily loaded, ill wrap it in this lazy component'.

Reply:
I hate the naming of it, its more just a loud flag to future selves. "Using a lazy artwork grid? this totally non-obvious thing has to be included".

Reply:
Where something like "Static" doesn't communicate the functional co-dependency -> Grid + LazyLoad + ... Static? Oh right: Grid + LazyLoad + LazyArtworkGrid. Mnemonic name hack

Reply:
Where something like "Static" doesn't communicate the functional co-dependency -> Grid + LazyLoad + ... Static? Oh right: Grid + LazyLoad + LazyArtworkGrid. Mnemonic name hack